src/App.tsx

```diff
@@ -15,36 +17,100 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+let goods = [...goodsFromServer];
+let isReverseCLicked = false;
```

Comment:
[CODE KNOWLEDGE] Avoid using global variables in React components. Instead, manage your data using React state.